Search/LinearSearch.js

```diff
@@ -7,15 +7,17 @@
 function SearchArray (searchNum, ar) {
   var position = Search(ar, searchNum)
   if (position !== -1) {
-    console.log('The element was found at ' + (position + 1))
+    console.log('The element was found at ' + (position + 1) + "th position")
   } else {
     console.log('The element not found')
   }
 }
 
 // Search “theArray” for the specified “key” value
 function Search (theArray, key) {
-  for (var n = 0; n < theArray.length; n++) {
+  //store theArray.length in a variable so that .length function is not summoned in every iteration 
```

Comment:
".length" is not a function, it is a value. Therefore, a function is not "summoned" in every iteration.